tools/getDepartures.php

```diff
@@ -0,0 +1,13 @@
+<form id="departuresForm">
+    <div class="input-group">
+        <button type="button" class="btn btn-primary" id="getLocation"><i class="fa-solid fa-location-crosshairs"></i> Get Location</button>
+    </div>
+</form>
+
```

Comment:
add a spacer / row here so the table and the button does not stick together.
![image](https://github.com/Relluem94s/RelluToolbox/assets/6181042/a542cf55-323c-45a7-8593-df229cdf4fbe)